FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <locus from="199" to="206"/> 
+                                        </item>
+                                        <item xml:id="q29" n="29">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="207" to="214"/> 
+                                        </item>
+                                        <item xml:id="q30" n="30">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="vii" to="x"/> 
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="intact">
+                                    <locus target="#48 #79 #109v #163v #208 #210 #211"/> blank. 
+                                    Few instances of watershed and dirt on various folios. Holes in <locus target="#26 #58 #112 #124 #138 #146"/> without textloss.
+                                    Repairs with a thread on <locus target="#196 #205"/>, Intentionally erased and mostly illegible text on <locus target="#210r #210v"/>, 
+                                    Unintentionally erased and partially illegible text on <locus target="#212r #212v"></locus>. Backside of a leaflet attached on the 
+                                    back side of the first, not numbered protective leaf is not recorded in the given images.
+                                </condition>
+                            </supportDesc>
+                            <layoutDesc><layout columns="1" writtenLines="11">
+                                <note>Number of written lines may vary from 9 to 15 on <locus from="1r" to="171r"/> and <locus from="175r" to="214v"/> 
+                                    but carrying 16 to 18 lines on <locus from="171v" to="174v"/>.</note>
+                                <ab type="pricking">Pricking is partly visible</ab>
+                                <ab type="ruling">Ruling is visible</ab>
+                            </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <ab type="script"/>
+                                <locus from="1r" to="171r"/><locus from="175r" to="207v"></locus>
+                                <seg type="script">Careful and angular, more fine and more tall than the following hands.</seg>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Incipits and sometimes sacred names and numbers as well as the name of the owner, that was always erased.</seg>
+                                <date from="1400" to="1500" cert="medium" datingMethod="paleography"/>
```

Comment:
I do not know datingMethod="paleography" - did I miss it? Is it somewhere in the guidelines?

Reply:
No, I did not find it in the guideline, but invented it by myself. I think, I used this way of indicating the reason also in other records before.

Reply:
it is very dangerous to use creative mark up and invent things not documented in the guidelines. please do not use it, i think I had corrected it in your other records before. we do not use `datingMethod`. but we provide `evidence="lettering"` in `origDate `for manuscripts dated palaeographically
if you believe something is missing in the guidelines and schema and must be introduced please open an issue, if discussed and approved then changes must first be implemented in the schema, then in the app to understand the schema, and only then can be used in the encoding.

Reply:
I am sorry for that! I used this method deliberately, because I thought, it Oxygen does not tell me, it won't be a such a dangerous mistake. I will try to change the record to the demanded method and I will look up the other records, that I have worked on so far.

Reply:
Done for BMLor403.

Reply:
I checked also the other record, that I have encoded to far. I found the same mistake in BMLor403. I think, I have cured it.